bundles/binding/org.openhab.binding.bticino/src/main/java/be/devlaminck/openwebnet/MonitorSessionThread.java

```diff
@@ -28,6 +28,9 @@
  * @author Tom De Vlaminck, Lago Moreno, Andrea Carabillo
  * @serial 1.0
  * @since 1.7.0
+ *
+ * @author Reinhard Freuis
+ * @since 1.11.0*
```

Comment:
No new `since` tags should be added.

Reply:
Ok
see commit https://github.com/openhab/openhab1-addons/pull/5486/commits/47252c5a1e77a4100415b334f458758822d8f760